sdk/cosmosdb/cosmos/src/queryExecutionContext/parallelQueryExecutionContextBase.ts

```diff
@@ -292,11 +299,14 @@ export abstract class ParallelQueryExecutionContextBase implements ExecutionCont
     } catch (err: any) {
       if (ParallelQueryExecutionContextBase._needPartitionKeyRangeCacheRefresh(err)) {
         // Split has happened so we need to repair execution context before continueing
+        this.err = new CosmosException(
+          `Split has happened so we need to repair execution context before continueing ${err}`
```

Comment:
this seems to be an internal comment (for developer readability purposes). Why are we exposing it as error message to the end user?